local_settings_example.py

```diff
@@ -469,6 +471,25 @@
     SAML_DJANGO_USER_MAIN_ATTRIBUTE = 'username'
     SAML_DJANGO_USER_MAIN_ATTRIBUTE_LOOKUP = '__iexact'
 
+    # The list of IdPs we talk to are defined
+    SAML_IDP = {
```

Comment:
Since this is not directly taken in by djangosaml2, I'd say let's make this a lower case variable.